portal/src/texts/profile/colors.md

```diff
@@ -5,81 +5,67 @@ order: 3
 
-## Farger i digitale løsninger
-I digitale løsninger bruker vi primærfargene og meldingssfargene. Illustrasjonsfargene reserverer vi til illustrasjoner og grafikk.
+## Grå- og vardevalører. 
+Vi har et spenn av valører for grå og varde. Disse kan vi bruke til å skape nyanser og nivåer i løsningene våre. Vi bruker disse i hovedsak bare på digitale flater.
 
-## Tilgjengelighet
-
-Tilgjengelighet er det viktigste å tenke på når vi bruker fargene i digitale løsninger. Vi skal alltid følge kravene til universell utforming. For farger betyr det [AA-kravet for kontrast](lenke). Kontrastkravet gjelder ikke bare tekst på bakgrunn, men også komponentenes kontrast mot sin egen bakgrunn, når det har betydning for å forstå funksjonaliteten.
-
-I tillegg til å tenke på tekst mot knappebakgrunnen, må vi for eksempel også tenke på knappebakgrunnen mot sidens bakgrunnsfarge.
-
-Tekstfargene våre er helhvit eller svart. Se på fargebeskrivelsene for å finne ut hvilken farge du skal bruke mot hvilken bakgrunn, for å overholde kontrastkravet.
-
-## Primærfarger
-
--   **Svart #000000**
-    Svart er en av Fremtinds profilbærende farger. Den bruker vi gjerne i elementer vi vil fremheve. Et godt eksempel er den svarte primærknappen. Vi kan bruke store, svarte flater, men de bør ha lite innhold i forhold til størrelsen. Unngå å sette svarte flater nær bakgrunner med fargen varde.
+### Gråvalører (Grå 10 - Grå 100)
+Disse kan brukes som bakgrunnsfarge på mindre flater og elementer for å bryte opp layout. Ikke bruk grå flater sammen med Varde, og vær obs på kontrast til innholdet når du bruker de ulike valørene. 
 
--   **Hvit #FFFFFF**
-    Hvit er også en av Fremtinds profilbærende farger. Den bruker vi som bakgrunnsfarge, og som tekstfarge på mørke flater. I tillegg er den _fokusfarge_, for eksempel på kort som er opphøyd over annet innhold, eller på inndataelementer som brukeren har valgt, for eksempel et felt i et skjema.
+Grå 10 til Grå 50 kan brukes med svart tekst, mens Grå 60 til Grå 100 kan brukes med hvit tekst.
 
--   **Varde #DED4C7**
-    Varde er vår siste profilbærende farge. Den bruker vi som bakgrunnsfarge eller på elementer, for å fremheve innhold. Når vi velger å bruke den på en flate, unngår vi å ha mye innhold på den, i forhold til størrelsen. Bruk svart på teksten, for å sikre god kontrast. Ikke bruke varde på flater som står nærme grå bakgrunner og elementer, eller svarte flater.
+### Vardevalører (Varde 10 - Varde 100)
+De lysere vardevalørene (Varde 50 og lavere) kan brukes som bakgrunnsflater eller på elementer for å bryte opp layout. Bruk de mørkere vardevalørene (Varde 60 og opp) med omhu. Ikke benytt de aller mørkeste som bakgrunnsfarge eller farge på større flater.
 
--   **Matt svart #3D3D3D**
-    Vi kan bruke matt svart som en mørkere bakgrunnsfarge. Som bakgrunnsfarge kan den ha hvit tekst.
+For Varde 10 til Varde 90 skal man bruke svart tekst.
```

Comment:
I de nye inputfeltene brukes Grå-60 over Varde-20 for placeholdertekst.

Reply:
Den bør sjekkes kontrastmessig - men ja, tror du det er vits å skrive ned tilfeller hvor vi bryter retningslinjene (vi burde isåfall ha et eget punkt om farge på placeholdertekst tenker jeg)

Reply:
Ganske sikker på at flere av placeholder-variantene feiler på kontrast. Spørsmålet er hvor viktig det er for akkurat den informasjonen. Bør som Mikkel sier dokumenteres godt hvis vi velger å bryte med standarden!

Reply:
https://www.w3.org/WAI/tutorials/forms/instructions/

Tilsier vel at placeholder ikke er unlatt å møte kontrastkravet?

Reply:
jeg forstår det som at placeholder bør møte krav. Da bør den i såfall bumpes opp til Grå-70.

Reply:
Bumpet til grå 70 😁